checks/azure_devops.go

```diff
@@ -40,30 +39,22 @@ func (t *AzureDevopsChecker) check(ctx *context.Context, check v1.AzureDevopsChe
 	var personalAccessToken string
 	if check.PersonalAccessToken.ValueStatic != "" {
 		personalAccessToken = check.PersonalAccessToken.ValueStatic
+	} else if connection, err := ctx.HydrateConnectionByURL(check.ConnectionName); err != nil {
+		return results.Failf("failed to hydrate connection: %v", err)
+	} else if connection != nil {
+		personalAccessToken = connection.Password
```

Comment:
I don't think this will ever execute

Reply:
It'll execute if the connection name is empty.